.gitmodules

```diff
@@ -375,3 +375,6 @@
 [submodule "contrib/antlr4-cpp-runtime"]
 	path = contrib/antlr4-cpp-runtime
 	url = https://github.com/antlr/antlr4.git
+[submodule "contrib/simdcomp"]
```

Comment:
(Most people add new submodules to the end.. If you want to avoid merge conflicts, better move the new lines a bit up.